REPOSITORY-GUIDELINES.md

```diff
@@ -0,0 +1,84 @@
+
+These repositories have varying levels of maturity and activity level. Discretion is advised for adoptors seeking to use any end user facing repositories in this scope.
+
+You can find the list of all ```Ecosystem``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#ecosystem-scope).
+
+### ```Historical```
+
+Historical repositories are considered deprecated, are no longer maintained, and are converted to an archived, read-only state. They are not recommended for use by adoptors.
+
+Repositories that have been inactive for quite some time may be good candidates for archival. They can be proposed through the same process as other [changes of scope](#change-of-scope).
+
+You can find the list of all ```Historical``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#historical-scope).
+
+## Lifecycle
+
+### Addition
+
+New repositories of any type can be proposed by using the [New Repository Template]() and can be approved via the process as described in [Approval process](#approval-process). This includes both donated repositories or the creation of new repositories.
+
+In order for a new repository to be approved, it must also meet the following criteria:
+
+- Approved by at least two committers
+- README describes the intent of the repository and has usage examples
+- Adheres to one of the [approved licenses](https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md)
+
+Any new repository must be added into the appropriate list in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md).
+
+### Change of scope
+
+If a repository's level of activity significantly changes and/or it modifies its purpose over time, it may become appropriate to pursue a change in scope. This change can be proposed by using the [Change of Scope Template]() and can be approved via the process as described in [Approval process](#approval-process).
+
+Any approved change of scope must be reflected in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md).
+
+### Approval process
+
+All changes that occur within the lifecycle of a repository have the same approval process:
+
+- An issue is opened via the appropriate issue template ([New Repository Template]() or [Change of Scope Template]())
+- Once the issue is opened, ample opportunity for discussion among both committers and the larger Cilium community is provided (for example, 1 week)
+- If there are no objections among committers and any relevant criteria are met (for example, the criteria found in [Addition](#addition)), the change is approved
+- If there are objections among committers but they can be worked through amicably, and any relevant criteria are met, the change is approved
```

Comment:
We could probably reorder these two and simplify the "objections" case: Usually when it comes to any proposals in the Cilium project, we don't assume the proposal is perfect the first time and either gets accepted or rejected. Rather, someone opens a proposal, then there is a comment period, which may lead to some discussion and changes, then ultimately that discussion leads the proposal to the final result. With that in mind:
```suggestion
- If the discussion suggests an alternate path should be pursued, then that discussion should be resolved prior to approval. 
- If there are no objections among committers and any relevant criteria are met (for example, the criteria found in [Addition](#addition)), the change is approved
```

Then the "rare case" below clearly defines what happens if the above steps do not result in a conclusive decision.